source/agora/node/Validator.d

```diff
@@ -360,4 +362,12 @@ public class Validator : FullNode, API
             this.enroll_man.updateRevealDistance(this.ledger.getBlockHeight());
         }
     }
+
+    /// Ditto
```

Comment:
What do you mean `/// Ditto`? That means `shutdown` becomes part of the `checkRevealPreimage` documentation because it's the previous symbol. Please document the function properly.